src/main/java/com/lenis0012/bukkit/marriage2/listeners/PlotSquaredListener.java

```diff
@@ -1,21 +1,21 @@
 package com.lenis0012.bukkit.marriage2.listeners;
 
-import com.intellectualcrafters.plot.PS;
-import com.intellectualcrafters.plot.object.Plot;
 import com.lenis0012.bukkit.marriage2.MData;
 import com.lenis0012.bukkit.marriage2.events.PlayerDivorceEvent;
 import com.lenis0012.bukkit.marriage2.events.PlayerMarryEvent;
+import com.plotsquared.core.PlotSquared;
+import com.plotsquared.core.plot.Plot;
 import org.bukkit.event.EventHandler;
 import org.bukkit.event.EventPriority;
 import org.bukkit.event.Listener;
 
 import java.util.UUID;
 
 public class PlotSquaredListener implements Listener {
-    private final PS plotSquared;
+    private final PlotSquared plotSquared;
 
     public PlotSquaredListener() {
-        this.plotSquared = PS.get();
+        this.plotSquared = PlotSquared.get();
```

Comment:
This is good, but I would like PlotSquared 5.x and 4.x support.

Reply:
We could do a Class.forName check on PS.java and PlotSquared.java, and use a different listener for each version.

Reply:
That might be one way to do it. The only thing I dislike about supporting PlotSquared 4.x is the necessary inclusion of a binary package into the git repository, or having to host a maven repository for it and any of it's presumably outdated dependencies.

Reply:
I honestly have not had the time to address this. Some of the required changes I can agree to, though I do believe it's about time to sunset the legacy server support.